src/common/formulas/functions.ts

```diff
@@ -221,6 +221,26 @@ const functions = [
     "str",
     () => firstNames[Math.floor(Math.random() * firstNames.length)]
   ),
+  new FormulaFunction<[{ name: "start", type: "str", optional: true }, { name: "end", type: "str", optional: true }], "str">(
+    "IPV4",
+    [{ name: "start", type: "str", optional: true }, { name: "end", type: "str", optional: true }],
+    "str",
+    (args, _ctx: FormulaContext) => {
```

Comment:
Would be good to follow the rest of the functions, stylistically. Params each on their own line, using destructing for `args`, no need for the generic parameters on line 224, using "as const" on the params for better type checking

Reply:
Also if you're not using the context arg, you can just omit it instead of prefixing with _